### DISCUSSION TITLE
strange Symfony Environment

### DISCUSSION THREAD

**Discussion by mattmarr06** [pimcore/pimcore#16584] [General]
In the backend ("Settings/Cache/Clear Cache/") I recently have the following entries.

"Symfony Environment: de_"
"Symfony Environment: dev"

or

"Symfony Environment: pro_"
"Symfony Environment: prod"

Where do the entries with "_" come from?

In the .env file I have the entry "APP_ENV=dev".

I'm really at a loss!

Installed is: Pimcore 10.6.9 / PHP 8.1 / 10.6.14-MariaDB 

Thank you for your help.

**Comment by skygatewitt** [comment]
I know this is an old thread, but in case someone comes across this problem: It happens when Symfony encounters a critical error during cache warmup. For me it was caused by this issue in Pimcore 10:  https://github.com/pimcore/pimcore/pull/15999

I applied this patch to fix it in 10.6.9: https://github.com/pimcore/pimcore/commit/dac19b99299cc1f57f71ea77837583eb50fbdfe7.patch
EDIT:
https://github.com/orgs/pimcore/discussions/16006#discussioncomment-8696560 
` "conflict": { "league/csv": ">=v9.11" },` fixes this as well.

Calling cache clear without the --no-warmup option creates the underscored directory, fails and never deletes the old cache. This also means that any changes made to the system.yml (aka "System Settings" in Admin UI) or potentially the database.yml will NOT take effect until the cache directory is deleted manually or by using the --no-warmup option.
